### ISSUE TITLE
Sun & directional light do not match-up.

### ISSUE THREAD

**Issue by Neopallium** [JonahPlusPlus/bevy_atmosphere#28]
Here is a patch to the `examples/cycle.rs` file to show the issue.  Shadows are turned on to help see the different angle of the directional light and the rendered Sun.

```
diff --git a/examples/cycle.rs b/examples/cycle.rs
index e52a5f1..4ca88ce 100644
--- a/examples/cycle.rs
+++ b/examples/cycle.rs
@@ -56,6 +56,10 @@ fn setup_environment(
     // Our Sun
     commands
         .spawn_bundle(DirectionalLightBundle {
+            directional_light: DirectionalLight {
+                shadows_enabled: true,
+                ..default()
+            },
             ..Default::default()
         })
         .insert(Sun); // Marks the light as Sun
@@ -94,7 +98,7 @@ fn setup_environment(
     // Spawn our camera
     commands
         .spawn_bundle(Camera3dBundle {
-            transform: Transform::from_xyz(5., 0., 5.),
+            transform: Transform::from_xyz(5., 0., 5.).looking_at(Vec3::new(0., 1., 0.), Vec3::Y),
             ..default()
         })
         .insert(AtmosphereCamera(None)) // Marks camera as having an atmosphere that isn't on a specific render layer

```

**Comment by Neopallium** [comment]
It seems the `looking_at` on the camera isn't causing the problem, since removing it doesn't sync-up the Sun and directional light.  Still need to have shadows on to easily see the direction of the light vs the rendered Sun.

This issue might be related to the change to default value for `sun_position` in the 0.4.0 release:
```
-            sun_position: Vec3::new(1.0, 1.0, 1.0),
+            sun_position: Vec3::new(0.0, 1.0, 1.0),
```

**Comment by JonahPlusPlus** [comment]
Thanks for pointing this out. It should be fixed with 58d5ccf12c9120a985d6413ecb5cf6e0f6c8120a.
